webscraper.js

```diff
@@ -35,12 +39,42 @@ if (require.main === module) {
                                             };
         }
 
-        return doc_dimensions;
-      }).then(function(dimensions) {
+        setTimeout(function () {
+           callback({
+              html: src,
+              css: node_properties,
+              folder : dir
+           });
+        });
+
+      }, item).then(function(doc_info) {
```

Comment:
I'm pretty sure, I pass in item to the function (which just has variable name dir), which I pass into the callback